lua/super-commit/git/commands.lua

```diff
@@ -1,20 +1,22 @@
 local M = {}
 
-function M.show_command_output(bufnum, command)
+function M.show_output(bufnum, command)
   local output_str =  vim.fn.system(command)
   local output_table = vim.split(output_str, '\n')
   vim.api.nvim_buf_set_lines(bufnum, 0, -1, false, output_table)
 end
 
-function M.show_diff_cmd_output(bufnum, path)
-  M.show_command_output(bufnum, "git diff --cached ".. path)
+function M.show_diff(bufnum, path)
+  M.show_output(bufnum, "git diff --cached ".. path)
 end
 
-local diff_keymap_msg = "To show git diff of files, select the file path in the window below, and press Enter."
-                        
-M.commands = {}
-M.commands[2] = "echo " .. diff_keymap_msg
-M.commands[3] = "git status"
-M.commands[4] = "git diff --name-only --cached"
+local file_sel_suggestion = "To show git diff of files, " ..
```

Comment:
It is up to you, but the word `sel` is ambiguous.
```suggestion
local file_select_suggestion = "To show git diff of files, " ..
```

Reply:
Changed variable name to `file_select_suggestion` in 788bd3aad0d2e8c32752314bf3200ea78e0ab325.

Reply:
Split the commits to 76e8f034e7c6e3f62571a8f1258a2cc23081edf3 and 788bd3aad0d2e8c32752314bf3200ea78e0ab325.